js/src/dht/findpeer.js

```diff
@@ -38,10 +38,9 @@ module.exports = (createCommon, options) => {
     after((done) => common.teardown(done))
 
     it('should find other peers', (done) => {
-      nodeA.dht.findpeer(nodeB.peerId.id, (err, peer) => {
+      nodeA.dht.findpeer(nodeB.peerId.id, (err, res) => {
```

Comment:
Documentation or implementation for `dht.findpeer` needs to be updated - `res` is not a `PeerInfo` instance.

https://github.com/ipfs/interface-ipfs-core/blob/master/SPEC/DHT.md#dhtfindpeer

Reply:
updated